hack/install.sh

```diff
@@ -105,6 +105,16 @@ rpm_import_repository_key() {
 	rm -rf "$tmpdir"
 }
 
+semverParse() {
+	RE='[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)'
+	#MAJOR
+	eval $2=`echo $1 | sed -e "s#$RE#\1#"`
+	#MINOR
+	eval $3=`echo $1 | sed -e "s#$RE#\2#"`
+	#MINOR
+	eval $4=`echo $1 | sed -e "s#$RE#\3#"`
```

Comment:
This could be simplified to use Bash features instead of relying so heavily on `sed`, ie something like:
```bash
major="${1%%.*}"
minor="${1#$major.}"
minor="${minor%%.*}"
patch="${1#$major.$minor.}"
patch="${patch%%[-.]*}"
```

```console
$ set -- 1.2.3-dev
$ major="${1%%.*}"
$ minor="${1#$major.}"
$ minor="${minor%%.*}"
14:34:49 [tianon@nameless ~]$ patch="${1#$major.$minor.}"
$ patch="${patch%%[-.]*}"
$ echo "$major" -- "$minor" -- "$patch" -- "$1"
1 -- 2 -- 3 -- 1.2.3-dev
```